ding/model/template/ppg.py

```diff
@@ -43,31 +72,68 @@ def __init__(
         self.aux_critic = copy.deepcopy(self.actor_critic.critic)
 
     def forward(self, inputs: Union[torch.Tensor, Dict], mode: str) -> Dict:
+        """
+        Overview:
+            Use different forward function according to mode.
+        Arguments:
+            - inputs (:obj:`Union[torch.Tensor, Dict]`): The input data.
+            - mode (:obj:`str`): The mode to forward.
+        Returns:
+            - output (:obj:`Dict`): The output data.
+        """
         assert mode in self.mode, "not support forward mode: {}/{}".format(mode, self.mode)
         return getattr(self, mode)(inputs)
 
     def compute_actor(self, x: torch.Tensor) -> Dict:
         """
+        Overview:
+            Use actor to compute action logits.
+        Arguments:
+            - x (:obj:`torch.Tensor`): The input data.
```

Comment:
don't use such simple description, you should show the meaning of tensor data or the common key-value in dict data

Reply:
fixed